Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -282,6 +282,8 @@ def identityToΓSpec : 𝟭 LocallyRingedSpace.{u} ⟶ Γ.rightOp ⋙ Spec.toLoc
         = toΓSpecFun X x := by rw [ContinuousMap.coe_mk]
       erw [this]
       dsimp [toΓSpecFun]
+      -- note to reviewers: this instance was found automatically before the refactor
+      have := @AlgebraicGeometry.LocallyRingedSpace.isLocalRingHomStalkMap X Y
```

Comment:
```suggestion
      -- TODO: this instance was found automatically before #6045
      have := @AlgebraicGeometry.LocallyRingedSpace.isLocalRingHomStalkMap X Y
```
could you please add a similar comment in all other places where a `have`/`let` is now needed?
Please include the number `#6045` in the comment, so that it in the future it will be easy to find all places where this happened.

Reply:
Sure! I've added comments to all places where a new `have` is required.